test/unary_test.js

```diff
@@ -736,4 +736,63 @@ describe('test unary', function() {
         ],
         [3, 2, 2, 1]);
   });
+
+  it('identity', function() {
```

Comment:
You may want to cover 0-D scalar input. @fdwr is working on the spec change and I am working a CL to Chromium. It would be good to have WPT test coverage as well. Thanks.

Reply:
> @fdwr is working on the spec change

I'm a little behind (got COVID Wednesday), but feeling better - I should continue Monday.

Reply:
No worries, glad to hear you are feeling better, have a good recovery first.